src/kit.ts

```diff
@@ -796,21 +798,21 @@ async function scanDirForClangForMSVCKits(dir: string, vsInstalls: VSInstallatio
         }
 
         const clangKits: Kit[] = [];
-        vsInstalls.forEach(vs => {
+        vsInstalls.forEach(async vs => {
             const install_name = vsDisplayName(vs);
             const vs_arch = (version.target && version.target.triple.includes('i686-pc')) ? 'x86' : 'amd64';
 
             const clangArch = (vs_arch === "amd64") ? "x64\\" : "";
-            const clangKitName = `Clang ${version.version} ${clang_cli} for MSVC ${vs.installationVersion} (${install_name} - ${vs_arch})`;
-            if (binPath.startsWith(`${vs.installationPath}\\VC\\Tools\\Llvm\\${clangArch}bin`) &&
-                util.checkFileExists(util.lightNormalizePath(binPath))) {
```

Comment:
util.checkFileExists(util.lightNormalizePath(binPath) was never awaited for, so it was always true. I put an "await" on it, but it gets stuck searching for clang.exe files.

Reply:
I think we should validate the path before adding the Kit. You can show me what you're seeing in our next meeting.

Reply:
It could actually be that `.forEach` want a sync function.  Convert it to a regular for loop and I think it will be better.
```ts
  for (const vs of vsInstalls) {
    ...
  }
```